csaf_files/IT/white/2025/va-25-136-01.json

```
@@ -1,192 +1,205 @@
-            "generator": {
-                "engine": {
-                    "name": "VINCE-NT",
-                    "version": "1.9.0"
+        {
+          "number": "1.0.0",
+          "summary": "Initial publication",
+          "date": "2025-05-16T20:14:51Z"
+        }
+      ]
+    },
+    "distribution": {
+      "tlp": {
+        "label": "WHITE"
+      }
+    },
+    "notes": [
+      {
+        "text": "All information products included in [https://github.com/cisagov/CSAF/tree/develop/csaf_files/IT/white](https://github.com/cisagov/CSAF/tree/develop/csaf_files/IT/white) are provided \\\"as is\\\" for informational purposes only. The Department of Homeland Security (DHS) does not provide any warranties of any kind regarding any information contained within. DHS does not endorse any commercial product or service, referenced in this product or otherwise. Further dissemination of this product is governed by the Traffic Light Protocol (TLP) marking in the header. For more information about TLP, see [https://us-cert.cisa.gov/tlp/](https://us-cert.cisa.gov/tlp/).",
+        "title": "Legal Notice",
+        "category": "legal_disclaimer"
+      },
+      {
+        "text": "Worldwide",
+        "title": "Countries and Areas Deployed",
+        "category": "other"
+      },
+      {
+        "text": "Information Technology",
+        "title": "Critical Infrastructure Sectors",
+        "category": "other"
+      },
+      {
+        "text": "IBM Security Guardium 11.5 is vulnerable to stored cross-site scripting (XSS). An attacker with administrative privileges could store arbitrary content or script in the administrative web interface that would be rendered or executed by users visiting modified parts of the web interface. The attacker could modify the web interface or obtain session tokens which could be used to log in as a different user.\n\n, which could allow a privileged user to disclose otherwise protected credentials. ",
+        "title": "Risk Evaluation",
+        "category": "summary"
+      },
+      {
+        "text": "Apply Guardium Data Protection patch 11.0p565 for Guardium 11.5.",
+        "title": "Recommended Practices",
+        "category": "general"
+      },
+      {
+        "text": "United States",
+        "title": "Company Headquarters Location",
+        "category": "other"
+      }
+    ],
+    "references": [
+      {
+        "url": "https://raw.githubusercontent.com/cisagov/CSAF/develop/csaf_files/IT/white/2025/va-25-136-01.json",
+        "summary": "Vulnerability Advisory VA-25-136-01 CSAF",
+        "category": "self"
+      }
+    ]
+  },
+  "product_tree": {
+    "branches": [
+      {
+        "category": "vendor",
+        "name": "IBM",
+        "branches": [
+          {
+            "category": "product_name",
+            "name": "Security Guardium",
+            "branches": [
+              {
+                "category": "product_version_range",
+                "name": "<11.5 11.0p565",
```

Patch identifier should not be included in the version range.
```suggestion
                "name": "<11.5",
```